driver/catalogue.c

```diff
@@ -48,7 +48,7 @@ static SQLRETURN fake_answer(SQLHSTMT hstmt, const char *src, size_t cnt)
 {
 	char *dup;
 
-	if (! (dup = strdup(src))) {
+	if (! (dup = _strdup(src))) {
```

Comment:
This caters to Microsoft’s pedantry, but will make porting to Linux harder. I think an alternative is to `/D_CRT_NONSTDC_NO_DEPRECATE`.